src/intl/messages.js

```diff
@@ -272,6 +272,12 @@ export const messages = {
   hdBoot: 'Hard Disk',
   networkBoot: 'Network (PXE)',
   cdromBoot: 'CD-ROM',
+  snapshot: 'Snapshots',
+  snapshotsTooltip: 'VM snapshots.',
+  noSnapshots: 'no snapshots',
+  newSnapshot: 'New snapshot',
+  addNewSnapshot: 'Add new snapshot',
+  areYouSureYouWantToDeleteSnapshot: 'Are you sure you want to delete snapshot {description}?',
```

Comment:
optional: Please have some sympathy for our translators and rename `description` to `snapshotName`. They usually don't have any context but the variable name.